### PR TITLE
Simplify regex that is used when parsing references

### PR DESCRIPTION
**Pull request by lexmag**

### CODE REVIEW

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

File: lib/gettext/po/parser.ex [hunk 1]
```diff
@@ -60,7 +60,7 @@ defmodule Gettext.PO.Parser do
 
   defp parse_references("#:" <> comment) do
     regex = ~r/
-      ((?:\w|\.|:)(?:\w|\s|\.|:|\/)+?)
+      ([\w\.:][\w\s\.:\/]+?)
```
Comment (lexmag) [inline lib/gettext/po/parser.ex]: I suppose we also should use `*` instead of `+` to allow `<one-letter>:<line>` references, right?

Comment (whatyouhide) [reply lib/gettext/po/parser.ex]: Wait, this regex doesn't work with the one letter ref? Boy I hate regexes, I find it so hard to parse this in my head 😂

Comment (lexmag) [reply lib/gettext/po/parser.ex]: It does not. 😁

Comment (whatyouhide) [reply lib/gettext/po/parser.ex]: @lexmag, you're my only hope. Just fix the regex however you see fit and add a test 😂

Comment (lexmag) [reply lib/gettext/po/parser.ex]: Done in #221.
